fiscalsim_us/variables/gov/states/id/tax/income/id_income_tax_before_refundable_credits.py

```diff
@@ -14,4 +14,4 @@ def formula(tax_unit, period, parameters):
             "id_income_tax_before_non_refundable_credits", period
         )
         non_refundable_credits = tax_unit("id_non_refundable_credits", period)
-        return max_(before_non_refundable_credits - non_refundable_credits, 0)
```

Comment:
@sydneybeutler. I don't think you want to make this change. The `max_()` format is correct. It is a policyengine-core function.